webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "requestHeaders": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "86"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "header": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "86"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "operation": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "86",
+                    "notes": "The <code>&quot;append&quot;</code> operation is supported from Chrome 108."
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "value": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "86"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
+          },
+          "responseHeaders": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "86"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
```

Comment:
`16`

Reply:
```suggestion
                  "version_added": false
```

Reply:
@Rob--W do you think it would make more sense to document `ModifyHeaderInfo` as an "extra" object under `RuleAction`

Reply:
Since `ModifyHeaderInfo` is not a real type, but a convenient way to document common interface, with different levels of supported properties, we may as well has separate BCD entries for `requestHeaders` and `responseHeaders`, and link them from the BCD in the `ModifyHeaderInfo` article.